src/pages/projectSettings/tabs/NotificationsTab/types.ts

```diff
@@ -1,42 +1,57 @@
+import { Unpacked } from "types/utils";
 import { ProjectType } from "../utils/types";
 
+export type Notification = Partial<{
+  notificationSelect: string;
+  jiraCommentInput: string;
```

Comment:
(Sorry idk where to put this comment):

We disallow this type of notification combo in the backend (i.e. task level JIRA comments because it can totally take down JIRA, although it doesn't look like this validation is being used correctly on the legacy page 🤔 ) which may be why we're seeing this error but it's not super helpful to users.
![image](https://user-images.githubusercontent.com/26798134/178596149-d7c4da3b-fe26-42ca-b30c-4578826a8d74.png)